index.bs

```diff
@@ -445,12 +450,14 @@ Every {{Document}} has a <dfn for="Document">post-prerendering activation steps
 
   1. Assert: |predecessorBC| is a [=top-level browsing context=].
 
-  <!-- The following are copied from the navigate algorithm, and probably could benefit from some refactoring to deduplicate. -->
+  <!-- The following except for the step that sets the |activation start time| are copied from the navigate algorithm, and probably could benefit from some refactoring to deduplicate. -->
 
   1. Cancel any preexisting but not yet [=navigate/mature=] attempts to navigate |predecessorBC|, including canceling any instances of the [=fetch=] algorithm started by those attempts. If one of those attempts has already <a lt="create and initialize a Document object">created and initialized a new `Document` object</a>, [=Document/abort=] that {{Document}} also.
 
   1. [=Prompt to unload=] the [=active document=] of |predecessorBC|. If the user [=refused to allow the document to be unloaded=], then return.
 
+  1. Let |activation start time| be the [=coarsened shared current time=] given the |predecessorBC|'s [=active document=]'s [=relevant settings object=]'s [=environment settings object/cross-origin isolated capability=].
```

Comment:
I think you want to use [current high resolution time](https://w3c.github.io/hr-time/#dfn-current-high-resolution-time) here, to make the timestamps relative to the time origin.

Reply:
I misread `activation start time` as setting the document's one rather than a separate local variable. Apologies! 
(maybe rename it to make the distinction clearer?)

Reply:
Renamed `activation start time` local variable to `coarsened time`.